### PR TITLE
[#IR-114] Add ordering in all list APIs

### PR DESCRIPTION
**Pull request by chetanyakan**

### CODE REVIEW

**Review by anmolghosh — CHANGES_REQUESTED** [review]
- Re-discuss the fields to be used for ordering
- Also re-check order for each API separately
- Check if any API requires composite ordering

**Comment by ishugupta92** [comment]
@anmolghosh Updated PR with updated ordering. Please Review.

File: apps/retrospective/services/retrospective_feedback.go [hunk 1]
```diff
@@ -246,6 +247,7 @@ func (service RetrospectiveFeedbackService) ListGoal(userID uint, sprintID strin
 	if err := query.
 		Preload("Assignee").
 		Preload("CreatedBy").
+		Order("resolved_at DESC, added_at DESC, created_at DESC").
```
Comment (anmolghosh) [inline apps/retrospective/services/retrospective_feedback.go]: this should be on basic of goal tyhpe

File: apps/retrospective/services/sprint.go [hunk 2]
```diff
@@ -550,7 +550,7 @@ func (service SprintService) GetSprintsList(retrospectiveID string, userID uint)
 		Where("(sprints.status <> ? OR created_by_id = ?)", retroModels.DraftSprint, userID).
 		Scopes(retroModels.NotDeletedSprint).
 		Preload("CreatedBy").
-		Order("end_date desc").
+		Order("end_date DESC").
```
Comment (anmolghosh) [inline apps/retrospective/services/sprint.go]: re-visit this, on basis of end date

File: apps/retrospective/services/sprint.go [hunk 3]
```diff
@@ -606,6 +607,7 @@ func (service SprintService) GetSprintMemberList(sprintID string) (sprintMemberL
 		Where("sprint_id = ?", sprintID).
 		Joins("JOIN users ON users.id = sprint_members.member_id").
 		Select("sprint_members.id, users.email, users.first_name, users.last_name, users.active").
+		Order("users.first_name, users.last_name").
```
Comment (anmolghosh) [inline apps/retrospective/services/sprint.go]: append id

File: apps/user/services/team.go [hunk 4]
```diff
@@ -23,7 +23,8 @@ func (service TeamService) UserTeamList(userID uint, onlyActive bool) (teams *us
 	filterQuery := db.Model(&userModels.Team{}).
 		Joins("JOIN user_teams ON teams.id = user_teams.team_id").
 		Where("user_teams.user_id = ?", userID).
-		Where("teams.active = true")
+		Where("teams.active = true").
+		Order("teams.name")
```
Comment (anmolghosh) [inline apps/user/services/team.go]: append created_At desc

**Review by anmolghosh — DISMISSED** [review]

File: apps/retrospective/services/retrospective_feedback.go (hunk 1 above)
Comment (ishugupta92) [reply apps/retrospective/services/retrospective_feedback.go]: Done

File: apps/retrospective/services/sprint.go (hunk 2 above)
Comment (ishugupta92) [reply apps/retrospective/services/sprint.go]: Done

File: apps/user/services/team.go (hunk 4 above)
Comment (ishugupta92) [reply apps/user/services/team.go]: Done

**Review by anmolghosh — APPROVED** [review]

**Pull request merged**